### DISCUSSION TITLE
How can I increase the number of HLOD Tile of IMDL format ?

### DISCUSSION THREAD

**Discussion by namsancow** [iTwin/itwinjs-core#234] [Q&A] [answered]
the big size model and the small size model have the same HLOD Tile size of 8 segments when they are opened by Desktop-Starter.
So, I guess the number of HLOD Tile is fixed as 8 segments regardless of model size.
For this reason, the small size model displaying speed is good and the freezing issue is not occurred. 
For this reason, the big size mode displaying speed slows down and the freezing issue becomes activated. 
How can I increase the number of HLOD Tile of IMDL format for increasing display speed and freezing issue?
Is there any control option in TileTree.ts, iModelTileTree.ts, iModelTree.ts, ImdlReader.ts ?

**Comment by pmconne** [comment, marked as answer]
Can you clarify what you mean by "freezing issue" please?
The application has no control over tile sub-division. It is unclear whether/how the sub-division relates to the "freezing issue".

**Reply by namsancow** [reply]
> Can you clarify what you mean by "freezing issue" please?
> The application has no control over tile sub-division. It is unclear whether/how the sub-division relates to the "freezing issue".

Dear Paul
Many thanks for kind response.
I solved my problem as commented in #169 
Thank you again for your kind advice.
